.travis.yml

```diff
@@ -58,6 +58,8 @@ env:
         - CONF="--enable-schedule-scalable"
         - CONF="--enable-dpdk-zero-copy"
         - CONF="--disable-static-applications"
+        - CONF="--disable-host-optimization"
+        - CONF="--disable-host-optimization --disable-abi-compat"
```

Comment:
As noted, this can be achieved by cross-compiling to an otherwise non-supported architecture.

Reply:
Cross-compiling can easily become troublesome, as we also need cross-OpenSSL, cross-CUnit, etc.

Reply:
Ok, I rescind that comment. Please fix the copyright dates and I'm Ok with this.

Reply:
@Bill-Fischofer-Linaro fixed now